<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoOptionsPage</class>
 <widget class="QWidget" name="VideoOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="6" column="0">
    <widget class="QLabel" name="descLabel">
     <property name="text">
      <string>Pipeline:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPlainTextEdit" name="descPlainTextEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="displayVideoCheckBox">
     <property name="text">
      <string>Display video</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="displayControlsCheckBox">
     <property name="text">
      <string>Display controls</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPlainTextEdit" name="infoPlainTextEdit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>Info:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="autoStartCheckBox">
     <property name="text">
      <string>Auto Start</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="respectAspectRatioCheckBox">
     <property name="text">
      <string>Respect aspect ratio</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="helpButton">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
